docs/source/gettingstarted.rst

```diff
@@ -5,9 +5,9 @@ Getting Started
 This page describes how to get started with AequilibraE.
 
 .. note::
-   Although AequilibraE is under intense development, you don't need to expect
-   important API changes, but you should check for new features often.
-
+   Although AequilibraE is under intense development, we try to avoid making
```

Comment:
I like it: "we try" :)

Reply:
A little kindness to oneself has never killed anyone... ;-)